sfxclient/gometrics.go

```diff
@@ -59,3 +65,47 @@ func (g *goMetrics) Datapoints() []*datapoint.Datapoint {
 		Gauge("num_goroutine", dims, int64(runtime.NumGoroutine())),
 	}
 }
+
+func (g *goEvents) Events() []*event.Event {
```

Comment:
other than symmetry with datapoints, what does this do? i feel like this isn't needed by anything.